test/pummel/test-net-connect-econnrefused.js

```diff
@@ -15,7 +15,7 @@ pummel();
 function pummel() {
   console.log('Round', rounds, '/', ROUNDS);
 
-  for (var pending = 0; pending < ATTEMPTS_PER_ROUND; pending++) {
+  for (let pending = 0; pending < ATTEMPTS_PER_ROUND; pending++) {
```

Comment:
Actually, as is the test failed (it's not run in CI), to fix I had to move the let to the previous line.